### ISSUE TITLE
y-axis label for the left-side bar plot

### ISSUE THREAD

**Issue by HyprValent** [jnothman/UpSetPlot#210]
I'm currently using this package in order to produce some images that will be used by others for analysis. 
In order to make it more understandable as a standalone image, I was looking into setting up a basic y-axis label on the leftside bar plot to explain what it means. Something like this:

![image](https://user-images.githubusercontent.com/32852290/212780846-cb16e24d-6a93-4a98-a443-c0ef847b8e47.png)

Is this currently possible in the package? I tried looking through the docs and couldn't find anything.
Thank you!

**Comment by jnothman** [comment]
Try something like:
```py
my_upset = UpSetPlot(...)
...
axes_dict = my_upset.plot(data)
totals_axes = axes_dict["totals"]
totals_axes.set_ylabel("Total observations")
```

**Comment by Xparx** [comment]
This example does not work for me,
`upsetplot.__version__`
`'0.8.0' `

```
import upsetplot
example = upsetplot.generate_counts()
axes_dict = upsetplot.plot(example)
totals_axes = axes_dict["totals"]
totals_axes.set_ylabel("blabla")
```
generates the figure
![image](https://user-images.githubusercontent.com/715716/227579291-6dad7c15-fcf1-4963-b9cf-6ea58e20077b.png)

For some reason the label isn't drawn(?).
I'm kind of guessing it might be the `sharey` setting that is spooking it.
as  `totals_axes.set_xlabel("blabla")` seems to work as expected.

**Comment by jnothman** [comment]
I've got a fix coming in 0.9.0, fixed in #243. In the meantime you'll need to use something like

```
my_upset = UpSetPlot(...)
...
axes_dict = my_upset.plot(data)
totals_axes = axes_dict["totals"]
label_axis = totals_axes.yaxis
label_axis.set_visible(True)
label_axis.set_ticklabels([])
label_axis.set_ticks([])
label_axis.set_label("Total observations")
```

**Comment by jnothman** [comment]
Actually, that's not working. We need a [more substantial change](https://github.com/jnothman/UpSetPlot/issues/259) to set a ylabel on totals plot. But you can set an xlabel fine.
